File: 프로그래머스/0/120808. 분수의 덧셈/분수의 덧셈.js
function solution(numer1, denom1, numer2, denom2) {
    var answer = [];
    var num = denom1*numer2 + denom2*numer1;
    var denom = denom1*denom2;
    const gcd = (a, b) => (a % b === 0 ? b : gcd(b, a % b));
    const result = gcd(num, denom); 
    answer[0] = num/result;
    answer[1] = denom/result;
    return answer;
}

File: 프로그래머스/0/120808. 분수의 덧셈/분수의 덧셈.py
import math

def solution(numer1, denom1, numer2, denom2):
    num = denom1*numer2 + denom2*numer1
    denom = denom1*denom2
    gcd = math.gcd(num, denom)
    return [num//gcd, denom//gcd]